test/e2e/Tests/Tests/RewindOrchestratorTests.cs

```diff
@@ -65,34 +68,53 @@ public async Task RewindFailedOrchestration_ShouldSucceed(int numFailures)
     {
         // Try to rewind a completed, running, terminated, and pending orchestration - all should fail
         HttpResponseMessage response = await HttpHelpers.InvokeHttpTrigger(
             "HttpStart_RewindOrchestration",
-            $"?orchestrationName=RewindParentOrchestration&input=complete&numFailures=0");
+            $"?input=complete&numFailures=0&callEntities=false");
         Assert.Equal(HttpStatusCode.Accepted, response.StatusCode);
         string instanceId = await DurableHelpers.ParseInstanceIdAsync(response);
         string statusQueryGetUri = await DurableHelpers.ParseStatusQueryGetUriAsync(response);
         await DurableHelpers.WaitForOrchestrationStateAsync(statusQueryGetUri, "Completed", 30);
 
         // Rewind a completed orchestration
         HttpResponseMessage rewindResponse = await HttpHelpers.InvokeHttpTrigger("RewindInstance", $"?instanceId={instanceId}");
-        Assert.Equal(HttpStatusCode.PreconditionFailed, rewindResponse.StatusCode);
+        // For all of the following tests, since Node throws a generic error in the case of a failure to rewind there is no great way 
+        // to return specific status codes, whereas .NET isolated returns specific error types which can be used to return specific status codes.
+        // So, in the Node case, we simply check for the BadRequest status code.
+        if (this.fixture.functionLanguageLocalizer.GetLanguageType() == LanguageType.DotnetIsolated)
+        {
+            Assert.Equal(HttpStatusCode.PreconditionFailed, rewindResponse.StatusCode);
+        }
+        else
+        {
+            Assert.Equal(HttpStatusCode.BadRequest, rewindResponse.StatusCode);
+        }
         response.Dispose();
         rewindResponse.Dispose();
 
+        // Rewind a running orchestration
         response = await HttpHelpers.InvokeHttpTrigger(
             "HttpStart_RewindOrchestration",
-            $"?orchestrationName=RewindParentOrchestration&input=run&numFailures=0");
+            $"?input=run&numFailures=0&callEntities=false");
         Assert.Equal(HttpStatusCode.Accepted, response.StatusCode);
         instanceId = await DurableHelpers.ParseInstanceIdAsync(response);
         statusQueryGetUri = await DurableHelpers.ParseStatusQueryGetUriAsync(response);
         await DurableHelpers.WaitForOrchestrationStateAsync(statusQueryGetUri, "Running", 30);
-
-        // Rewind a running orchestration
         rewindResponse = await HttpHelpers.InvokeHttpTrigger("RewindInstance", $"?instanceId={instanceId}");
-        Assert.Equal(HttpStatusCode.PreconditionFailed, rewindResponse.StatusCode);
+        if (this.fixture.functionLanguageLocalizer.GetLanguageType() == LanguageType.DotnetIsolated)
+        {
+            Assert.Equal(HttpStatusCode.PreconditionFailed, rewindResponse.StatusCode);
+        }
+        else
+        {
+            Assert.Equal(HttpStatusCode.BadRequest, rewindResponse.StatusCode);
+        }

```

Comment:
Why different response values depending on the language? Should we add comments explaining this?